JeremiesLeftEye

```diff
@@ -0,0 +1,6 @@
+cornea
+some melanine 
+eyelashes
+
```

Comment:
Hey, i would really like to know the color of jeremie's eye. Could you add this in?

Reply:
brown!